annotationframeworkclient/annotationengine.py

```diff
@@ -352,7 +202,12 @@ def get_table_metadata(self, table_name: str, aligned_volume_name=None):
         url = self._endpoints["table_info"].format_map(endpoint_mapping)
 
         response = self.session.get(url)
-        return handle_response(response)
+        d = handle_response(response)
```

Comment:
please use a more informative variable than instead of `d`

Reply:
fixed!